recipe/meta.yaml

```diff
@@ -24,8 +23,8 @@ requirements:
   run:
     - aiohttp >=3.3.1
     - aioitertools >=0.5.1
-    - botocore >=1.23.24,<1.23.25
-    - python >=3.6
+    - botocore >=1.27.59,<1.27.60
+    - python
     - wrapt >=1.10.10
 
 test:
```

Comment:
Please remove `python <3.10` from test/requires.

Reply:
This has been removed